dyninstAPI/src/inst-aarch64.C

```diff
@@ -642,31 +642,51 @@ Register EmitterAARCH64::emitCall(opCode op,
         assert(reg!=REG_NULL);
     }
 
-    //instPoint *point = gen.point();
-    //assert(point);
-    assert(gen.rs());
-
-    //Address of function to call in scratch register
-    Register scratch = gen.rs()->getScratchRegister(gen);
-    assert(scratch!=REG_NULL);
-    gen.markRegDefined(scratch);
-    insnCodeGen::loadImmIntoReg<Address>(gen, scratch, callee->addr());
+    // if (gen.func()->obj() != callee->obj())
+    // {
+    //     // assert(0); // inter module call
+
+    //     // l: not using emitPLTCommon for now
+    //     emitPLTCall(callee, gen);
+    // }
+    // else
+    // {
+
+        //instPoint *point = gen.point();
+        //assert(point);
+        assert(gen.rs());
```

Comment:
I think this block should be out-dented since the outer if/else is not being used.

Reply:
Fixed, I was mainly concerned about messing up the history...